streams/src/main/java/org/apache/kafka/streams/state/internals/ChangeLoggingWindowBytesStore.java

```diff
@@ -62,7 +66,7 @@ public void put(final Bytes key, final byte[] value) {
     public void put(final Bytes key, final byte[] value, final long timestamp) {
         if (key != null) {
             bytesStore.put(key, value, timestamp);
-            changeLogger.logChange(WindowStoreUtils.toBinaryKey(key, timestamp, 0, innerStateSerde), value);
+            changeLogger.logChange(WindowStoreUtils.toBinaryKey(key, timestamp, maybeUpdateSeqnumForDups(), innerStateSerde), value);
         }
     }
 
```

Comment:
super nit: I know this pre-existed, but IMHO line 77 a little tough to read what about
```
innerStateSerde = getStateSerdes(context.applicationId(), bytesStore.name());
....
private StateSerdes<Bytes, byte[]> getInnerStateSerdes(String appId, String storeName) {
  return WindowStoreUtils.getInnerStateSerde(ProcessorStateManager.storeChangelogTopic(appId, storeName));
}

Reply:
could just inline the calls so it is all on one line?